### DISCUSSION TITLE
Do you think train GNN model with some test information(I mean like if training_loss be found much higher than the test_loss then use strategies like minbatch) is permitted?

### DISCUSSION THREAD

**Discussion by Kevin-shihello-world** [pyg-team/pytorch_geometric#6184] [Ideas]
As the question. I mean just sometimes it's hard to decide the validation sets' size so if we set a threshold of training_acc minus testing_acc and use some strategies to deal with the training, would this be permitted by the related society? (I'm an undergraduate student)
Thanks!

**Comment by rusty1s** [comment]
Can you clarify what you mean? In general, you are allowed to make use of node features coming from the validation and test set during training (transductive learning), but you definitely shouldn't use any test labels.
